Dangerfile

```diff
@@ -45,11 +45,8 @@ def check_pr_size_increase
   bypass_size_check = github.pr_labels.any? { |s| s == BYPASS_LABEL }
 
   if bypass_size_check
-    warn("Size check is being bypassed due to the presence of the label \"#{BYPASS_LABEL}\"")
     if total_size_increase > WARN_SIZE_INCREASE
```

Comment:
I would be ok tweaking the warning/failure size increases thresholds with this if we want to be more conservative though.